interface/Geometry2023.h

```diff
@@ -788,6 +788,9 @@ class HPK_W11_22_3_20T_500x500_150M_C600_PixelsGeometry : public DefaultGeometry
     std::vector<double> positionRecoParCol = {0.250000, -1.063382, 12.004629, -103.303789, 376.826502, -487.471421};
     std::vector<std::vector<double>> xSlices = {{0.70, 0.90}, {1.20, 1.40}, {1.65, 1.85}}; // Raw data coordinates
     std::vector<std::vector<double>> ySlices = {{-0.65, -0.45}, {-1.15, -0.95}}; // Raw data coordinates
+    // std::vector<std::vector<double>> sensorEdges = {{-0.70,-0.50}, {0.70, 0.50}}; //square interior of pads
+    // std::vector<std::vector<double>> sensorEdgesExtra = {{-0.70,-0.50}, {0.70, 0.50}}; //square interior of pads
+    // std::vector<std::vector<double>> sensorEdgesTight = {{-0.70,-0.50}, {0.70, 0.50}}; // Sensor's local frame
```

Comment:
I'd leave the sensorEdgesTight info there as long as it's useful for a study, such as the one-strip plots we discussed. If not, let's remove the commented lines!